config/routes.rb

```diff
@@ -1,3 +1,8 @@
 Rails.application.routes.draw do
+  devise_for :users
+  resources :events
+  root 'events#index'
+  get '/users', to: 'users#show'
+  resources :attended_events
```

Comment:
please use proper flags for each resources, for example:
      resources :attended_events, only: %i[index]